include/mega/megaclient.h

```diff
@@ -224,6 +224,9 @@ class MEGA_API MegaClient
     // queue a user attribute retrieval (for non-contacts)
     void getua(const char* email_handle, const attr_t at = ATTR_UNKNOWN, int ctag = -1);
 
+    // retrieve the email address of a user
+    void getUserEmail(const char *uid = NULL);
```

Comment:
I don't think it's the case. The `uid` is mandatory for the command, AFAIK, so better to not default the value.